app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  validates :name, presence: true
+  validates :username, presence: true
+  validates :username, uniqueness: true
```

Comment:
this could be written like this:
`validates :username, presence: true, uniqueness: true`